### PR TITLE
Create Redis maps to speed up the lookups

### PR DESCRIPTION
**Pull request by p-janik**
Staging is very slow. I believe this should speed it up a bit. We need to trigger createMaps endpoint once and then maps would be used.

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -771,6 +771,11 @@ router.post(
...
 );
 
+app.post(
+  "/__BACKOFFICE__/createMaps",
+  safeRequestHandler(backofficeAPI.createMaps)
+);
+
 // HEALTH CHECK
 app.get("/health", (req, res) => {
   res.status(200).send("there is no piwo tracker!");
```
Comment (lera) [inline src/app.ts:781]: no piwo? 😳

**Review by lera — APPROVED** [review]
let's try it!

File: src/db.ts [hunk 2]
```diff
@@ -263,7 +263,8 @@ export const getDevice = async (deviceId) =>
     )
   );
 
-export const getAllDevices = async () => {
+// DEPRECATED
```
Comment (UgurGumushan) [inline src/db.ts]: If you use @deprecated, IDEs recognize it and use strikethrough on usages
https://jsdoc.app/tags-deprecated.html

File: src/db.ts [hunk 3]
```diff
@@ -263,7 +264,8 @@ export const getDevice = async (deviceId) =>
...
```
Comment (p-janik) [inline src/db.ts]:
```suggestion
// @deprecated
```

**Review by UgurGumushan — APPROVED** [review]

**Pull request merged**